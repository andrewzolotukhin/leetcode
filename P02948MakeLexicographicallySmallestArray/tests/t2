6 3
1 7 6 18 2 1
