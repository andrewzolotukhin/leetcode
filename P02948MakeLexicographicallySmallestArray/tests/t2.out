1
6
7
18
1
2
